web-client/src/pages/admin/AdminInventoryPage.tsx

```diff
@@ -1,5 +1,100 @@
-import UnderConstruction from "@/components/UnderConstruction";
+import { Button } from "@/components/ui/button";
+import { Search } from "lucide-react";
+import { ChevronDown } from "lucide-react";
+import { DropdownMenu, DropdownMenuItem, DropdownMenuTrigger, DropdownMenuContent } from "@/components/ui/dropdown-menu";
+import InventoryTable from '@/components/InventoryTable';
+import { useNavigate } from 'react-router-dom';
+import { useGetSocks } from '@/api/inventory/queries';
+import { useState } from 'react';
 
 export default function AdminInventoryPage() {
-  return <UnderConstruction pageName="Inventory" />;
+  const navigate = useNavigate();
+
+  const [page, setPage] = useState(1);
+  const limit = 8;
+  const offset = (page - 1) * limit;
+
+  const { data, error, isLoading } = useGetSocks(limit, offset);
+
+  if (isLoading) {
+    return <div>Loading...</div>;
+  }
+
+  if (error) {
+    return <div>Error fetching socks: {error instanceof Error ? error.message : 'Unknown error'}</div>;
+  }
+
+  if (!data || !data.items || data.items.length === 0) {
+    return <div>No socks available in the inventory.</div>;
+  }
+
+  const handleRowClick = (sockId: string) => {
+    navigate(`/socks/${sockId}`);
```

Comment:
This should be `/admin/socks/${sockId}` since we want to keep admins in the admin side of the app

Reply:
I now get taken to this page everytime I click on a sock. Do I still add this change?
<img width="352" alt="image" src="https://github.com/user-attachments/assets/4df99749-c14c-4381-80e9-04110b41ce87">

Reply:
Yep that's fine, Bora is working on that page

Reply:
Fixed!